packages/lit-protocol-cipher/README.md

```diff
@@ -75,6 +86,9 @@ async function example() {
     await litProvider.disconnectWallet();
```

Comment:
Please remove this redundant command - superseded by `finally`

Reply:
done